app/controllers/amr_heatmap_controller.rb

```diff
@@ -42,21 +42,23 @@ def amr_counts
         end
       end
       amr_data << {
-        sample_name: sample.name,
-        sample_id: sample.id,
-        amr_counts: amr_counts,
-        error: ""
+        "sampleName" => sample.name,
```

Comment:
We seem to have a convention to pass snake_case fields back from Rails Controllers. Even though it leads to an awkward mix in React. I would follow convention and change to snake_case here, so we're at least consistent.

Reply:
I was going by https://github.com/chanzuckerberg/idseq-web/blob/master/DEV_GUIDELINES.md - see the last line.

Reply:
Talked to Tiago. This is good. You're totally right, I missed the dev guidelines.